src/main/java/com/orasi/bluesource/Employees.java

```diff
@@ -55,8 +56,10 @@ public void employeeSearch(String strSearch){
 	 * @author Paul
 	 */
 	public void clickAddEmployee() {
-		btnAdd.syncEnabled(5,true);
-		btnAdd.click();		
+		btnAdd.syncVisible(2,true);
+		btnAdd.syncEnabled(2,true);
+		btnAdd.syncInFrame(2,true);
+		btnAdd.click();
```

Comment:
Lot of syncs here....so only one of them should be needed.  Can you just do syncEnabled()?